handbook/product/design/design_process.md

```diff
@@ -121,15 +121,57 @@ While asynchronous communication is a core attribute of remote work, key moments
       - Schedule a meeting with engineers to discuss the interaction
   - Tools
       - Figma
-- Metrics collection:
+- **Design to engineering handoff**
+    - Design preperation
+        - Ensure colors utilize the named color styles vs. the color palette where possible 
+            - This adds our color style variable name to the code palette 
```

Comment:
I'm not sure what you mean here by 'this'

Reply:
I've clarified in the most recent version.